### ISSUE TITLE
Request for Fine-tuned Pythia Models or Training Code

### ISSUE THREAD

**Issue by Fuyujia799** [thu-wyz/inference_scaling#8]
Hello,

Thank for sharing this excellent work!

It seems that the reward model has been uploaded, but the fine-tuned Pythia models are missing. Though the code for fine-tuning is provided, I noticed that the code does not include the training configurations or details for fine-tuning Pythia models. Could you please either upload the fine-tuned Pythia models or provide the relevant training code and configurations for fine-tuning the Pythia models?

This would be very helpful for users trying to replicate the results.

Thank you!

**Comment by thu-wyz** [comment]
Hi, I am not able to opensource them now, but you can find the training configurations in the paper appendix C.
